packages/nocodb-sdk/src/lib/enums.ts

```diff
@@ -163,20 +163,20 @@ export enum WorkspacePlan {
 }
 
 export const RoleLabels = {
-  [WorkspaceUserRoles.OWNER]: 'Owner',
-  [WorkspaceUserRoles.CREATOR]: 'Creator',
-  [WorkspaceUserRoles.EDITOR]: 'Editor',
-  [WorkspaceUserRoles.COMMENTER]: 'Commenter',
-  [WorkspaceUserRoles.VIEWER]: 'Viewer',
-  [ProjectRoles.OWNER]: 'Owner',
-  [ProjectRoles.CREATOR]: 'Creator',
-  [ProjectRoles.EDITOR]: 'Editor',
-  [ProjectRoles.COMMENTER]: 'Commenter',
-  [ProjectRoles.VIEWER]: 'Viewer',
-  [ProjectRoles.NO_ACCESS]: 'No Access',
-  [OrgUserRoles.SUPER_ADMIN]: 'Super',
-  [OrgUserRoles.CREATOR]: 'Creator',
-  [OrgUserRoles.VIEWER]: 'Viewer',
+  [WorkspaceUserRoles.OWNER]: 'owner',
```

Comment:
@mustafapc19

Reply:
@reenphygeorge Any reason why u have converted all roles to lowercase for `RoleLabels` ?